docs/howtos/create_custom_appcharts.md

```diff
@@ -137,20 +141,81 @@ The necessary Kubernetes CRD is provided by the Epinio installation.
 Apply this resource to the Epinio cluster:
```

Comment:
Reminder as commented offline:
Probably good idea to reinforce the need of an external server where the app tarball exist

Reply:
Does not fit in the flow. I actually saw that we had a proper text about this at the beginning of the section, just not really highlighted. This is now a `caution` block.

Reply:
Done with commit 7115b3be85fb0438a6b20f14a45de84b2f93d5dc